src/miner/[minerId].md

```diff
@@ -0,0 +1,71 @@
+```js
+import { LineGraph } from "../components/line-graph.js";
+import { getDateXDaysAgo } from "../utils/date-utils.js";
+const data = FileAttachment(`../data/${observable.params.minerId}-spark-rsr-summary.json`).json();
+```
+
+<div class="hero">
+  <body><a href="/"><img src="../media/spark-logomark-blue-with-bbox.png" alt="Spark Logo" width="300" /></a><body>
+    <h2>Dashboard Beta</h2>
+    <body><a href="https://filspark.com/dashboard" target="_blank" rel="noopener noreferrer">(Click here for Legacy Spark Grafana Dashboard)</a><body>
+</div>
+
+<h4>Miner Spark RSR Summary</h4>
+<body>This section shows the miner Spark Retrieval Success Rate Score summary. You can adjust the date range. Records start on the 7th April 2024.</body>
```

Comment:
Nothing happens when I adjust the date range 🤔

Reply:
I think there's some strange behavior if there's no data for provider in certain range. 

If we compare these two providers [f010479](https://add-per-miner-rsr-visualizat.spark-dashboard.pages.dev/miner/f010479) and [f03156617](https://add-per-miner-rsr-visualizat.spark-dashboard.pages.dev/miner/f03156617) earliest entry in database for f010479 is on 7th April 2024 but f03156617 it's 29th July 2024.